app/views/koi/admin_crud/_form_field_inline.html.erb

```diff
@@ -70,7 +70,6 @@
         $(this).parent("h3").toggleClass("nested-fields-visible");
         $(this).parent("h3").parent(".nested-fields").siblings().find("h3").removeClass("nested-fields-visible").next(".nested-inline-fields").hide();
         $(this).closest(".nested-fields").children(".nested-inline-fields").toggle();
-        reInitCkEditor();
```

Comment:
Do we know why it was being reininitialized here in the first place? What do you think it was trying to solve / fix?